anime-list-master.xml

```diff
@@ -1467,7 +1467,7 @@
   <anime anidbid="349" tvdbid="77681" defaulttvdbseason="0" episodeoffset="" tmdbid="" imdbid="tt0161981">
     <name>Gekijouban Slayers</name>
     <mapping-list>
-      <mapping anidbseason="1" tvdbseason="0">;1-7;2-7;3-7;</mapping>
+      <mapping anidbseason="1" tvdbseason="0">;1-1;2-1;3-1;</mapping>
```

Comment:
Mapping multiple anidb episodes to one TVDB episode will not result in good metadata. Instead only map the first AniDB episode and map the others to 0.

Otherwise, if you have TVDB priority and 2 video files (part 1 and 2), both will have the same title "Slayers Perfect".

Reply:
Accounted for your changes, did not know this was a best practice, thank you for the clarity.